projects/igniteui-angular/src/lib/grids/api.service.ts

```diff
@@ -150,18 +150,14 @@ export class GridBaseAPIService <T extends IgxGridBaseDirective & GridType> {
             return args;
```

Comment:
Additional improvement wil be if we return here `ards.cancle` or just `true` and strongly type the return value of the method. We are using this value at two places:
1. In `cell.componente.ts` `update` method. Change `const args` with `const isCanceled` (or some better name) and use the new boolean variable in the `if` statement.
2.  In `grid-base.directive.ts` in `updateCell` method. You can change it in the same way as in #1.

Reply:
Actually, I really prefer not to change this, as returning the whole args collections is not that much of an overkill.